src/session_impl.cpp

```diff
@@ -898,7 +898,22 @@ namespace aux {
 		m_ses_extensions.push_back(ext);
 		m_alerts.add_extension(ext);
 		ext->added(session_handle(this));
+
+		dht_extensions_t dht_ext;
```

Comment:
would you mind commenting a bit on what's going on here? what are dht extensions, and extension dht queries?